packages/tss-client/src/client.ts

```diff
@@ -25,10 +21,6 @@ if (globalThis.js_read_msg === undefined) {
     const tss_client = globalThis.tss_clients.get(session) as Client;
     tss_client.log(`reading msg, ${msg_type}`);
     if (msg_type === "ga1_worker_support") {
-      // runs ga1_array processing on a web worker. Message processing is non-blocking, it is also sequential.
-      // TODO: Remove this web worker.
-      // There is no parallel benefit here. Additionally, removing this also will allow the
-      // dkls library to be used from tss-lib instead of being downloaded a second time from the server.
       return tss_client._workerSupported || "unsupported";
```

Comment:
```suggestion
      return "unsupported";
```

Also remove `_workerSupported`, since this PR strips the worker out.

Reply:
done